### PR TITLE
add ut of cache/v2/cluster.go

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**
/kind enhancement

**What this PR does / why we need it**:
add ut of cache/v2/cluster.go
**Which issue(s) this PR fixes**:
a part of #104

### CODE REVIEW

File: pkg/cache/v2/cluster_test.go [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+		}
+		cluster2 := &cluster_v2.Cluster{
+			ApiStatus:      core_v2.ApiStatus_UPDATE,
+			Name:           "ut-cluster2",
+			ConnectTimeout: uint32(60),
+			LbPolicy:       cluster_v2.Cluster_ROUND_ROBIN,
+		}
+		cache.SetApiCluster(cluster1.Name, cluster1)
+		cache.SetApiCluster(cluster2.Name, cluster2)
+		cache.Flush()
```
Comment (hzxuzhonghu) [inline pkg/cache/v2/cluster_test.go:62]: I mean we should know that it really does write bpf map.

Comment (LiZhenCheng9527) [reply pkg/cache/v2/cluster_test.go:62]: I think it's up to the UT of `map_v2.ClusterUpdate`, to test if it actually writes the bpf map

Comment (hzxuzhonghu) [reply pkg/cache/v2/cluster_test.go:62]: I mean according to the apiStatus, we should know whether this handler work as expected.  Am i clear?

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**